### DISCUSSION TITLE
[bug] Amazon Identity Provider does not accept scope = openid and Keycloak always sets it

### DISCUSSION THREAD

**Discussion by paolotateo** [keycloak/keycloak#35265] [Q&A]
Today I tried to setup "Login by Amazon" using the OpenID Connect v1.0 provider, but it didn't work as Amazon responds "An unknown scope was requested". 
Actually I compiled the configuration by inserting "Scopes = profile", which is an admissible value, but Keycloak sends the value "openid+profile". 
In the field tip I read "The scopes to be sent when asking for authorization. It can be a space-separated list of scopes. Defaults to 'openid'."
I guess I can call it a bug, as I should have the option to not send Amazon the value openid.

Does anyone know where the code to fix this is located?

Thanks in advance

**Comment by embesozzi** [comment]
It could be related to `application/x-www-form-urlencoded`. Therefore, `openid profile` is encoded as `openid+profile`

On the other hand, if your following the OpenID Connect standard you MUST use the `openid` scope.
